### DISCUSSION TITLE
a 225Mb exe for a KEYBOARD MAP? (Windows)

### DISCUSSION THREAD

**Discussion by Kratoklastes** [binarybottle/engram-layout#59] [General]
And if you dom;t trust whoever wrote the 'Keyman' bloatware, the engram map goes away when you uninstall.

A 4kb file **with a quarter-Gb .dependency**: what possible **innocuous** explanation is there for that?

My main machine runs Arch, BTW.

**Comment by sunaku** [comment]
You can build a native Windows keymap from [this tiny **9.4 KB** source file](https://github.com/binarybottle/engram/blob/a08a8be30a9fdb6e691795d794595676e54e4bb4/keymaps/windows/engram.klc) as follows:

1. Install the official _Microsoft Keyboard Layout Creator_ app: [MSKLC version 1.4](
  https://www.microsoft.com/en-us/download/details.aspx?id=102134
).

2. Launch the MSKLC app and load the `engram.klc` keyboard layout source file: 

       "C:\Program Files (x86)\Microsoft Keyboard Layout Creator 1.4\MSKLC.exe" engram.klc

3. Open the "Project" menu and select the "Build DLL and Setup Package" item.

4. Open the resulting build directory and run the `setup.exe` installer file.
